packages/web/src/pages/audio-rewards-page/components/modals/ChallengeRewardsModal/ChallengeRewardsModal.tsx

```diff
@@ -435,7 +436,7 @@ const ChallengeRewardsBody = ({ dismissModal }: BodyProps) => {
   }
 
   const renderProgressBar = () => {
-    if (showProgressBar) {
+    if (showProgressBar && challenge.max_steps) {
```

Comment:
if `challenge.max_steps` is part of `showProgressBar` do we still need it down here too?

Reply:
I tried that and TS couldn't pick up that it would not be null here.